br/cmd/br/main_test.go

```diff
@@ -75,3 +76,18 @@ func TestRunMain(*testing.T) {
 
 	<-waitCh
 }
+
+func TestCalculateMemoryLimit(t *testing.T) {
```

Comment:
Could we add a test case for invalid input, like -1024 or something. just in case 
`meqleft := memtotal - memused` returns a negative value,